pkg/nftables/nftables.tpl

```diff
@@ -82,3 +82,18 @@ table inet firewall {
 	}
 {{- end }}
 }
+{{- if .AdditionalDNSAddrs }}
+
+# Add additional DNS addresses for dnat redirection for the dns proxy
+table inet nat {
+	set public_dns_servers {
+		type ipv4_addr
+		flags interval
+		auto-merge
+		elements = {
+		{{- $sep := " " }}
+		{{- range .AdditionalDNSAddrs }}{{ $sep }}{{ . }}{{ $sep = ", " }}
```

Comment:
There seems to be `StringsJoin` available.

Reply:
Thanks, used that.